--- microcode_ctl-1.08/microcode_ctl.c.orig	2001-04-15 00:18:13.000000000 +0200
+++ microcode_ctl-1.08/microcode_ctl.c	2004-08-30 23:24:43.701959736 +0200
@@ -19,15 +19,6 @@
 #include <errno.h>
 #include <unistd.h>
 #include <string.h>
-#include <linux/version.h>
-
-#if LINUX_VERSION_CODE < KERNEL_VERSION(2,3,0)
-#include <asm/types.h>
-struct task_struct { int a ; };
-#endif
-
-#include <asm/cache.h>
-#include <asm/processor.h>
 
 static char *progname;
 int print_normal_messages=1;
